launch/test/launch/substitutions/test_command.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2020 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests the Command substitution."""
+
+import os
+import pathlib
+
+from launch.launch_context import LaunchContext
+from launch.substitutions import Command
+from launch.substitutions.substitution_failure import SubstitutionFailure
+
+import pytest
+
+this_dir = pathlib.Path(__file__).parent
+
+commands = {
+    'normal': str(this_dir / 'test_command' / 'normal_command.bash'),
+    'failing': str(this_dir / 'test_command' / 'failing_command.bash'),
+    'with_stderr': str(this_dir / 'test_command' / 'command_with_stderr.bash')
+}
+
+if os.name == 'nt':
+    for key, value in commands.items():
+        commands[key] = value.replace('bash', 'bat')
```

Comment:
@ivanpauno nit: consider using `pytest` fixtures instead of global variables.

Reply:
Done